src/prefect/server/api/root.py

```diff
@@ -11,3 +13,18 @@
 async def hello():
     """Say hello!"""
     return "👋"
+
+
+@router.get("/ready")
```

Comment:
`GET /api/ready`

i toyed with adding this to the base router, but i think that `/api/ready` is the correct endpoint namespace bc we're really checking the API server's ability to draw data and respond to requests; we dont really care about the UI server in this situation

Reply:
A lot of other services in Kubernetes use the `z` suffix convention, maybe something we can consider (or maybe not, since we already have health) https://stackoverflow.com/questions/43380939/where-does-the-convention-of-using-healthz-for-application-health-checks-come-f

Reply:
Yeah, I'd just stick with the existing (in our app) convention.

Reply:
i knew that the `z` was something from google, but interesting to know more about the origin details